src/router/index.tsx

```diff
@@ -18,6 +19,9 @@ export default function Routes() {
         <Route path="/404">
           <PageNotFound />
         </Route>
+        <Route exact path="/sign-up">
```

Comment:
```suggestion
        <Route path="/sign-up">
```

Actually, we don't need `exact` property for every path except the default one (`/`)